app/src/main/java/com/babylon/wallet/android/presentation/wallet/WalletScreen.kt

```diff
@@ -291,6 +291,10 @@ private fun WalletAccountList(
                 )
                 Spacer(modifier = Modifier.height(RadixTheme.dimensions.paddingXLarge))
             }
+        } else {
+            item {
+                Spacer(modifier = Modifier.height(RadixTheme.dimensions.paddingXLarge))
+            }
```

Comment:
No spacer was added when we have no fiat value. Making the cards appear very close to the title, so we add the same spacer added on top of fiat value when visible.